.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this file could be ignored in `.gitignore` since it's a personal toolchain configuration.